templates/admin/snaps.html

```diff
@@ -7,9 +7,24 @@
 {% block admin_content %}
 <div class="l-application__header">
   <h2 class="p-heading--four">Snaps in {{ store.name }}</h2>
-  <div>
-    <label for="snaps-filter" class="u-off-screen">Filter snaps</label>
-    <input type="search" name="snaps-filter" placeholder="Search" data-js-snaps-filter>
+  <div style=" align-items: space-between; display: flex;">
+    <div style="margin-right: 1rem;">
+      <label for="snaps-filter" class="u-off-screen">Filter snaps</label>
+      <input type="search" class="u-no-margin--bottom" name="snaps-filter" placeholder="Search" data-js-snaps-filter>
+    </div>
+    <button class="p-button--neutral u-no-margin--bottom" data-js-open-side-panel-btn>
```

Comment:
you could do:
`<button class="p-button--neutral u-no-margin--bottom" data-js-open-side-panel-btn><i class="p-icon--plus">Add</i><span>Add snap</span></button>`

Reply:
@solazio thanks - fixed